src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2896,6 +2904,30 @@ namespace winrt::TerminalApp::implementation
         ShowWindowChanged.raise(*this, args);
     }
 
+    winrt::fire_and_forget TerminalPage::_SearchMissingCommandHandler(const IInspectable /*sender*/, const Microsoft::Terminal::Control::SearchMissingCommandEventArgs args)
+    {
+        assert(!Dispatcher().HasThreadAccess());
+
+        if (!Feature_QuickFix::IsEnabled())
+        {
+            co_return;
+        }
+
+        std::vector<hstring> suggestions;
+        suggestions.reserve(1);
+        suggestions.emplace_back(fmt::format(L"winget install {}", args.MissingCommand()));
```

Comment:
Oops. Yeah, we're blocked by winget. Internally tracking [here](https://microsoft.visualstudio.com/OS/_workitems/edit/50066117).

Removed any references to the WinMD since we're not actually using it.